react/__tests__/components/ProductPrice.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ProductPrice from './../../ProductPrice'
+import { render } from 'enzyme'
+
+describe("<ProductPrice />", () => {
+  let renderComponent = null
```

Comment:
Why not instead of having a global variable, have a function that generate and returns the component to be tested?